app/code/Magento/Cookie/etc/frontend/di.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
```

Comment:
Module `Cookie` is not responsible for managing Session Cookies.
It's only purpose is to display a warning, that website uses cookies, when configured accordingly.